files/openafs-client-Ubuntu

```diff
@@ -33,6 +33,7 @@ MODULEDIR=${MODULEDIR:-$MODULEROOT/kernel/fs}
 DKMSDIR=${DKMSDIR:-$MODULEROOT/updates/dkms}
 AFS_KERNEL_MODULE=libafs
 OPENAFSDKMS=openafs-modules-dkms
+DEFAULTCELL="sunrise.ericsson.se"
```

Comment:
We will have soon 3 other AFS-cells in the GIC.
DEFAULTCELL is a fallback if the client is not able to get his AFS_cell where it belongs to.
But it's a good idea to use the AFS-Variable afs_cell here.

/Ralf


Am 23.02.2015 um 09:12 schrieb Phil Friderici <notifications@github.com<mailto:notifications@github.com>>:


In files/openafs-client-Ubuntu<https://github.com/Phil-Friderici/puppet-module-afs/pull/33#discussion_r25148379>:

> @@ -33,6 +33,7 @@ MODULEDIR=${MODULEDIR:-$MODULEROOT/kernel/fs}
>  DKMSDIR=${DKMSDIR:-$MODULEROOT/updates/dkms}
>  AFS_KERNEL_MODULE=libafs
>  OPENAFSDKMS=openafs-modules-dkms
> +DEFAULTCELL="sunrise.ericsson.se<http://sunrise.ericsson.se>"


same is true for the other osfamilies too

—
Reply to this email directly or view it on GitHub<https://github.com/Phil-Friderici/puppet-module-afs/pull/33/files#r25148379>.